templates/patterns/list_item/pattern-list-item.html.twig

```diff
@@ -53,6 +54,11 @@
               type: 'standalone',
               label: title,
               path: url
+            },
+            icon: {
```

Comment:
I still have doubts that it is safe. We rely on `ECL-link` ECL component implementation where we have to check that the icon name is not empty. But what if we will have a change in implementation. I don't sure that it is possible, but the chance still exists. It would be simpler to keep the empty icon element if we don't have the `icon` name from our side for explicitly declaring that link without an icon.